### PR TITLE
docs(ee/changelog) 0.33-1 release

### PR DESCRIPTION
**Pull request by gszr**
### Summary

Add changes/fixes present in our upcoming release, 0.33-1.

### CODE REVIEW

**Comment by coopr** [comment]
LGTM!

One possible redundancy to simplify (or not, if you want to be really explicit): 

```
**Kong EE 0.33** has these notices from **Kong CE 0.13**:
  - Support for **Postgres 9.4 has been removed** - starting with 0.32, Kong Enterprise does not start with Postgres 9.4 or prior
```

```
  - Additional requirements:
      - **Vitals** requires Postgres 9.5+
```

Well, if Kong won't start with Postgres 9.4 or prior, that means that Postgres will be 9.5 or higher for Kong to start - which means that Vitals has the Postgres it needs.

**Comment by gszr** [comment]
Good point, thanks @coopr.

File: app/enterprise/changelog.md [hunk 1]
```diff
@@ -11,6 +12,78 @@ nav:
...
+      - Remove dependency on "public" schema or "pg_default" tablespaces in Postgres - such a dependency would cause migrations to fail if such tablespaces weren't being used
+    - **Healthchecks**
+      - Fix Host header in active healthchecks
+      - Fix for connection timeouts on passive healthchecks
+  - **Dev Portal**
+      - Fix issue leading migrations `2018-04-10-094800_dev_portal_consumer_types_statuses` and `2018-05-08-143700_consumer_dev_portal_columns` to fail
+      - Automatically log users in after registering when auto-approve is on
+      - Reduce the size of the mobile breakpoint
+      - Improve performance and file sizes of bundled code
+      - 0.33 Portal shows the name of a spec twice after uploading
```
Comment (leevye) [inline app/enterprise/changelog.md]: Suggest changing to 'Fix issue causing Portal to list spec files twice'

**Comment by gszr** [comment]
Thank you @leevye, addressed your comments.

**Pull request merged**